lib/croppable/model.rb

```diff
@@ -42,7 +42,8 @@ def #{ name }=(croppable_param)
                   self.#{ name }_croppable_data = Croppable::Datum.new(croppable_param.data.merge(name: "#{ name }"))
                 end
 
-                to_crop_croppable[:#{ name }] = self.#{ name }_croppable_data.updated_at_previously_changed? || self.#{ name }_croppable_data.new_record?
+                self.#{ name }_croppable_data.updated_at_previously_changed? || self.#{ name }_croppable_data.new_record?
```

Comment:
There should be an `if` in here. If this line is true `to_crop_croppable` should be set